src/styleguide/mock-data/api-response/notification-response.js

```diff
@@ -0,0 +1,22 @@
+import notification from '@/styleguide/mock-data/data-object/notification';
+
+/**
+ * Creates an empty API response with a notification.
+ *
+ * @param {Object} data - The notification object.
+ *
+ * @returns {Object}
+ */
+function createApiResponse(data) {
+  return {
+    data: {},
+    messages: [data],
```

Comment:
Hm we could do it here in the template but anyway we have to adjust this for each Backend in the project. Because this property (and also the structure inside) gets defined by the backend system. Guess in Pimcore we're more flexible and we should be able to change it to `notifications`.